app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] The login page should match the design with the wireframe. Please update the devise login view.

### Your design 

![Screenshot from 2021-12-08 21-36-25](https://user-images.githubusercontent.com/22434660/145242459-a8f186d0-6e0f-46b3-9127-8adcd77a5949.png)

### Given design 

![Screenshot from 2021-12-08 21-38-17](https://user-images.githubusercontent.com/22434660/145242315-41364541-6812-45f7-a721-4db2299c881d.png)